src/src/pages/capabilities/capabilityTags/capabilityTagsSubForm.js

```diff
@@ -31,6 +46,48 @@ const CustomDropdown = function (props) {
   );
 };
 
+const CustomCheckbox = function (props) {
+  const { onChange } = props;
+  const [selectedOption, setSelectedOption] = useState(undefined);
+  let c1 = (Math.random() + 1).toString(36).substring(5);
+  let c2 = (Math.random() + 1).toString(36).substring(5);
+
+  useEffect(() => {
+    onChange(selectedOption);
+  }, [selectedOption]);
+
+  return (
+    <div>
+      <input
+        className={styles.checkBox}
+        id={"checkbox-" + c1}
+        name={"checkbox-" + c1}
+        type="checkbox"
+        checked={selectedOption === undefined ? false : selectedOption}
```

Comment:
think you can write this as `selectedOption ?? false` ?

Reply:
This is correct for this particular line, but for the remaining three using `!selectedOption` it is not possible, as inverting undefined is `true` which changes the logic.
I prefer that all 4 lines are similar, as it lowers mental load (I believe), but I will change this one line if you prefer it that way.